circuits/sha1.circom

```diff
@@ -67,6 +68,12 @@ template Sha1(nBits) {
     }
 
     for (k=0; k<160; k++) {
-        out[k] <== sha1compression[nBlocks-1].out[k];
```

Comment:
I would rather have it written this way:

```
    for (i=0; i<5; i++) {
        for (k = 0; k < 32; k++) {
            out[32*i + k] <== sha1compression[nBlocks-1].out[32*i + (31-k)];
        }
    }
```

Reply:
Changed as requested